lib/ocrypto/ec_key_pair_test.go

```diff
@@ -0,0 +1,74 @@
+package ocrypto
+
+import (
+	"testing"
+)
+
+func TestECKeyPair(t *testing.T) {
+	for _, modeGood := range []ECCMode{ECCModeSecp256r1, ECCModeSecp384r1, ECCModeSecp521r1} {
+		ecKeyPair, err := NewECKeyPair(modeGood)
+		if err != nil {
+			t.Fatalf("NewECKeyPair(%d): %v", modeGood, err)
+		}
```

Comment:
Just so our tests are consistent let's stick with the testify package for assert/require going forward.

```suggestion
		require.NoError(t,err, "message....")
```